examples/AxialStretchingCase/axial_stretching.py

```diff
@@ -87,6 +88,16 @@ class StretchingBeamSimulator(BaseSystemCollection, Constraints, Forcing, CallBa
     EndpointForces, 0.0 * end_force, end_force, ramp_up_time=1e-2
 )
 
+# add damping
+damping_constant = 1.0
+dl = base_length / n_elem
+dt = 0.01 * dl
+stretch_sim.dampen(stretchable_rod).using(
+    ExponentialDamper,
```

Comment:
Open for discussion. Do we wanna use `Damper`? Wouldn't it have the potential to mislead what it is? For example, we can have external `damper` implemented as force, etc. This is more `internal` feature, I feel like.

Reply:
I don't believe that `damping` is an internal feature since the original Cosserat rod equations don't have this term. Rather just like forces/torques, it's an external to-be documented feature (available and visible to the users) that's added to the simulator to numerically `soften` or stabilize the simulations. Additionally, the presence of documentation, which will be added soon by @armantekinalp  and I will further avoid any potential confusion that may arise. Finally, the damper you are referring to can be appropriately named as `DampingForce` and added through the `Forcing` class, thus avoiding any potential clash/confusion with this module.

Reply:
I agree with @bhosale2 on this point. Cosserat Rod does not have damping internally build in so having it as a separate  module makes more sense. Also with documentation it will be more clear the details and how to use `damping` module.